README.md

```diff
@@ -1,4 +1,8 @@
-# React-shop-cloudfront
+# Soccer Shop
+
+#URLS:
+- [Cloudfront](https://d1woy1337wrj09.cloudfront.net/)
```

Comment:
"Access denied" when trying to access pages using direct urls. For e.g.

https://d1woy1337wrj09.cloudfront.net/cart
https://d1woy1337wrj09.cloudfront.net/admin/products